nucliadb_protos/writer.proto

```diff
@@ -121,6 +121,8 @@ message BrokerMessage {
     resources.Extra extra = 37;
 
     repeated resources.FieldQuestionAnswerWrapper question_answers = 38;
+
+    utils.Security security = 39;
```

Comment:
Have you forgotten to add the `utils.proto` changes?

Reply:
No, the protos were part of a previous PR that was adding the functionality to the IndexNode only. For if you want to have a look to have the full picture of the feature:

https://github.com/nuclia/nucliadb/commit/4f6c44281ad32ac01d33d87963a2ef737fb50c6a